app/src/behaviors/behavior_sensor_rotate.c

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) 2020 The ZMK Contributors
```

Comment:
Same thing for updating the year of the copyright header as the devicetree binding (`.yaml`)